docs/source/install.rst

```diff
@@ -266,10 +266,10 @@ Create a build directory and step into it::
     mkdir build
     cd build
 
-From the build directory you can now configure CMake and build the binaries::
+From the build directory you can now configure CMake, build and install the binaries::
 
     cmake ..
-    cmake --build .
```

Comment:
Keep this second line in for the regular "build" step. The third new line does the "install" step (while it is confusing that it also has "build" in it too). Configure, build and install are three separate commands.

Reply:
Alright, third attempt. Not sure if it's the right way still. Sorry about this.